src/app/app-modules/nurse-doctor/doctor-worklist/doctor-worklist.component.html

```
@@ -1,301 +1,83 @@
-      </div>
-    </div>
-    <div class="col-xs-4 col-sm-6 col-md-8 fit_to_content">
-      <button
-        mat-raised-button
-        id="caseSheetButton"
-        style="display: none"
-        [disabled]="true"
-        class="btn btn-link pull-right">
-        {{ currentLanguageSet?.bendetails?.emergencyCaseSheet }}
-      </button>
+<app-beneficiary-worklist
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)